### DISCUSSION TITLE
Am constantly rejected my uploaded proof

### DISCUSSION THREAD

**Discussion by catsaysme0w** [community/community#139950] [GitHub Education]
### Hi everyone,

Hey guys,
I’m a high-school student from Israel (tenth grade) and I’m trying to get my student status approved.
As proof I’m uploading a photograph of a, obviously in Hebrew, school certification that I attend.
But I’am constantly rejected. Below is the latest rejection message.
By now I’ve spent several hours (over 4) dotting my i’s and crossing my t’s with regard to updating my 2FA, Profile, Billing information etc. I’ve tried with my name in English and conversely native Hebrew — to no avail, I always get rejected.
Any help will be appreciated, thanks.

> Hey Hila Shachar,
> 
> Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Sep 27, 2024 for [@catsaysme0w](https://github.com/catsaysme0w).
> 
> The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.
> Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.
> Please use your device camera to submit academic affiliation documents.
> Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). Variations of your name and nicknames are unacceptable.
> You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

**Comment by davevad93** [comment]
Hi @catsaysme0w ,

My advice (if you haven't already did so) is to upload the original academic documentation in your native language plus an English translated version, also make sure it contains a current date in a `DD/MM/YYYY`, `MM/DD/YYYY` or `YYYY/MM/DD` format and not just the academic year (e.g. `2024/2025`). For a successful application, the proof must contain your full name, the school's name and a current date.

For more information, check the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352):

- [The image you selected does not appear to contain your school name](https://github.com/orgs/community/discussions/111352#image-does-not-appear-to-contain-your-school-name)
